frontend/src/components/modules/Separator/Separator.tsx

```diff
@@ -28,11 +27,13 @@ export const Separator = ({ data }: SeparatorProps) => {
 		separatorConfigs.logoStart;
 
 	return (
-		<div className="separator">
-			<Animation
-				animationData={config.image}
-				height={config.height}
-				width={config.width}
+		<div className={`separator ${config.className}`}>
+			<Image
+				src={config.image.src}
+				alt="Combined AFŽ Logo and Background"
+				className="combined-separator-svg"
```

Comment:
Mislim da bi bolji naziv za ovu CSS kalsu bio `separator__image`. Deo `combined-separator` ne znam na šta se tačno odnosi, a `svg` je nešto što može biti zamenjeno u budućnosti sa `jpg`, `png` itd.